src/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -54,29 +57,61 @@ export default function VideoPlayer(
       ? `https://player.vimeo.com/video/${videoId}?autoplay=0&loop=0`
       : `https://www.youtube.com/embed/${videoId}?disablekb=1&autoplay=0&fs=1&modestbranding=0`;
 
+  const iFramTitleEmbedCode = iframeTitle ? `${iframeTitle}` : `Video player`;
+
+  const embedCodeFinal =
+    embedCode && embedCode.includes("<iframe") && !embedCode.includes("title=")
+      ? embedCode.replace(`<iframe `, `<iframe title="${iFramTitleEmbedCode}" `)
+      : embedCode;
+
   const errorMessage =
     "<strong>Error:</strong> This video player has not been configured properly. Please contact the site administrator.";
 
   let isInvalid = false;
-  if (!videoType && !videoId) {
-    console.warn("VideoPlayer requires the `videoType` and `videoId` props");
+  if (!embedCodeFinal && !videoType && !videoId) {
```

Comment:
Can you also add tests to make sure these warnings are logged to the console with missing props?